lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -4853,10 +4853,11 @@ swift::irgen::IRGenModule &SwiftASTContext::GetIRGenModule() {
 
         std::lock_guard<std::recursive_mutex> global_context_locker(
             IRExecutionUnit::GetLLVMGlobalContextMutex());
+        llvm::StringRef PrivateDiscriminator = "";
         m_ir_gen_module_ap.reset(new swift::irgen::IRGenModule(
             ir_generator, ir_generator.createTargetMachine(), nullptr,
             GetGlobalLLVMContext(), ir_gen_opts.ModuleName, PSPs.OutputFilename,
-            PSPs.MainInputFilenameForDebugInfo));
+            PSPs.MainInputFilenameForDebugInfo, PrivateDiscriminator));
```

Comment:
Shouldn't we pass in the Private discriminator of the current module? It's stored as an anonymous namespace in DWARF.

Reply:
Yes, this is what we should do.